CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ include(CTest)
 option(BUILD_RELOCATABLE_BINARIES
         "Build Relocatable Binaries, this will turn off features that will fail on older kernels than used for the build."
         OFF)
+option(BYO_CRYPTO "Don't build a tls implementation or link against a crypto interface. This feature is only for unix builds currently."
```

Comment:
Should w name it BYO_CRYPTO or BYO_TLS?

Reply:
I'm thinking of having an identical option in aws-c-cal so we can make sure the option controls both builds from a higher-level binding.